system/jhtree/jhtree.ipp

```diff
@@ -36,18 +36,21 @@ class CKeyStore
 {
 private:
     Mutex mutex;
-    Mutex idmutex;
     CKeyIndexMRUCache keyIndexCache;
-    int nextId;
-    int getUniqId() { synchronized procedure(idmutex); return ++nextId; }
-    IKeyIndex *doload(const char *fileName, unsigned crc, IReplicatedFile *part, IFileIO *iFileIO, IMemoryMappedFile *iMappedFile, bool isTLK, bool allowPreload);
+    std::atomic<unsigned> nextId = { 0x80000000 };
```

Comment:
Why the different initial value?  It seems to still be initialised to 0 in the constructor.

std::atomic<unsigned> nextId{ 0x80000000 };

is more common syntax.

Reply:
Different initial value is intended to avoid clashes between values allocated by the ccdfile cache and ones allocated outside it. I have removed the initialization to 0 in the constructor